### PR TITLE
Available Fields fixed

### PR DESCRIPTION
**Pull request by saniya16**
# Description
Added css in siglens.css
image

Fixes ##556 Fix CSS Style: Available Fields on Live Tail Screen

# Checklist:

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: static/live-tail.html [hunk 1]
```diff
@@ -25,6 +25,7 @@
     <link rel="stylesheet" href="./css/lib/all.min.css" />
     <link rel="stylesheet" href="./css/lib/ag-grid.min.css" />
     <link rel="stylesheet" href="./css/siglens.css" />
+
```
Comment (nbhavana) [inline static/live-tail.html:29]: remove the new line

**Review by nbhavana — COMMENTED** [review]
Lgtm. While you are in the same code, can you add a fix for the `Select/Unselect All` to be in the same line like this 
image

**Comment by saniya16** [comment]
image
Fixed

**Comment by saniya16** [comment]
Have fixed the issue

**Review by nbhavana — APPROVED** [review]
lgtm

**Pull request merged**